.jenkinsci-new/build.groovy

```diff
@@ -13,7 +13,7 @@ def cmakeConfigure(String buildDir, String cmakeOptions, String sourceTreeDir=".
 }
 
 def cmakeBuild(String buildDir, String cmakeOptions, int parallelism) {
-  sh "cmake --build ${buildDir} ${cmakeOptions} -- -j${parallelism}"
+  sh "cmake --build ${buildDir} ${cmakeOptions} -- -j${parallelism} | sponge buildTimeResult.txt"
```

Comment:
This fails on mac: `sponge: command not found`
you can change it to 

def cmakeBuild(String buildDir, String cmakeOptions, int parallelism, String pipe) {
sh "cmake --build ${buildDir} ${cmakeOptions} -- -j${parallelism}  ${pipe}"

where
  def pipe = '  | sponge buildTimeResult.txt '
or
  def pipe = ''